RequestService/RequestService.Repo/Helpers/QuestionExtenstions.cs

```diff
@@ -333,7 +340,8 @@ private static IEnumerable<SupportActivities> GetSupportActivitiesForRequestForm
             IEnumerable<SupportActivities> genericSupportActivities = Enum.GetValues(typeof(SupportActivities)).Cast<SupportActivities>()
                 .Where(sa => sa != SupportActivities.WellbeingPackage && sa != SupportActivities.CommunityConnector
                  && sa != SupportActivities.ColdWeatherArmy && sa != SupportActivities.Transport
-                 && sa != SupportActivities.Steward && sa != SupportActivities.FrontOfHouseAdmin && sa!= SupportActivities.HealthcareAssistant && sa != SupportActivities.BackOfficeAdmin);
+                 && sa != SupportActivities.Steward && sa != SupportActivities.FrontOfHouseAdmin && sa!= SupportActivities.HealthcareAssistant && sa != SupportActivities.BackOfficeAdmin
+                 && sa != SupportActivities.MealsOnWheels && sa != SupportActivities.VolunteerSupport && sa != SupportActivities.MealtimeCompanion);
```

Comment:
Again, this list is getting longer.  Maybe we create a list positively rather than negatively.